rq/cli/cli.py

```diff
@@ -267,7 +267,8 @@ def worker(
             queues,
             name=name,
             connection=cli_config.connection,
-            default_worker_ttl=worker_ttl,
+            default_worker_ttl=worker_ttl,  # TODO remove this arg in 2.0
+            worker_ttl=worker_ttl,
```

Comment:
Not sure if adding a `TODO` is part of the code style of this, feel free to remove it. I just wanted to highlight the change.

Reply:
This may be removed in version 3.0. I plan on releasing 2.0 sometime in the next few weeks so this would not give people enough time to adapt their code.